.github/workflows/update-markdown.yml

```diff
@@ -53,14 +49,22 @@ jobs:
         run: |
           python .github/scripts/sdk_doc.py
 
-      - name: Commit files
-        uses: ChromeQ/commit@v2.0.0
-        env:
-          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          files: |
-            cmd/commands.md
-            sdk/SDK.md
-          commit-message: "chore: Update MD files [skip ci]"
-          ref: ${{ github.head_ref }}
+      - name: Check if there are changes
```

Comment:
I can see that it sets 
`changes=true`  in $GITHUB_ENV

I am not sure what it does with this